_api-reference/ingest-apis/ingest-pipelines.md

```diff
@@ -0,0 +1,280 @@
+        }
+      }
+    }
+  ]
+}
+```
+
+### Query parameters 
+
+Query parameters are optional.
+
+Parameter | Type | Description
+:--- | :--- | :---
+`verbose` | Boolean | Verbose mode. Display data output for each processor in the executed pipeline.
+
+### Request body fields
+
+Field | Type | Description
+:--- | :--- | :---
+`pipeline` | Object | The pipeline to be simulated. If the pipeline identifier is not included, then the response simulates the latest pipeline created. Optional.
+`docs` | Array | The documents to be used to test the pipeline. Required.
+
+The `docs` field can include the following subfields:
+
+Field | Type | Description
+:--- | :--- | :---
+`id` | String | A unique identifier for a document. The identifier cannot be used elsewhere in the index. Optional.
+`index` | String | The index where the document's transformed data appears. Optional.
+`source` | Object | The document's JSON body. Required.
+
+## 
+
+### Receive a verbose response 
+
+When the `verbose` parameter is set to `true`, the response shows the processor results, that is, how the processor transformed the data. 
+
+```json
+{
+  "docs" : [
+    {
+      "processor_results" : [
+        {
+          "processor_type" : "set",
+          "status" : "success",
+          "doc" : {
+            "_index" : "index",
+            "_id" : "id",
+            "_source" : {
+              "field-name" : "value",
+              "location" : "document-name"
+            },
+            "_ingest" : {
+              "pipeline" : "35678",
+              "timestamp" : "2022-02-03T21:45:09.414049004Z"
+            }
+          }
+        }
+      ]
+    },
+    {
+      "processor_results" : [
+        {
+          "processor_type" : "set",
+          "status" : "success",
+          "doc" : {
+            "_index" : "index",
+            "_id" : "id",
+            "_source" : {
+              "field-name" : "value",
+              "location" : "document-name"
+            },
+            "_ingest" : {
+              "pipeline" : "35678",
+              "timestamp" : "2022-02-03T21:45:09.414093212Z"
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+
+## Get information about a pipeline
+
+Use the get ingest pipeline API operation to return information about a pipeline.
+
+## Path and HTTP methods
+
+The following example request returns information about a specific pipeline. 
+
+```json
+GET _ingest/pipeline/pipeline-id
+```
+{% include copy-curl.html %}
+
+The following example request returns information about all ingest pipelines.
+
+```
+GET _ingest/pipeline
+```
+{% include copy-curl.html %}
+
+The following is an example response.
+
+```json
+{
+  "pipeline-id" : {
+    "description" : "Optional description of the pipeline",
+    "processors" : [
+      {
+        "set" : {
+          "field" : "field-name",
+          "value" : "value"
+        }
+      }
+    ]
+  }
+}
+```
+
+## Delete a pipeline
+
```

Comment:
Processor documentation in progress. See [#4193](https://github.com/opensearch-project/documentation-website/issues/4193)